worlds/boxes.world.erb

```diff
@@ -23,8 +44,8 @@
 %>
 <sdf version="1.5">
   <world name="default">
-    <physics type="ode">
-      <max_step_size>0.001</max_step_size>
+    <physics type="<%= engine %>">
```

Comment:
we don't technically need to encode the physics engine in the world file for `boxes.cc` because we can specify the physics engine by name when loading the world; these other parameters are good though